## CPSR annotation resources

### Basic variant consequence annotation
  * [VEP v94](http://www.ensembl.org/info/docs/tools/vep/index.html) - Variant Effect Predictor ([GENCODE v28](https://www.gencodegenes.org/releases/28.html) as gene reference database (v19 for grch37))

###  *Insilico* predictions of effect of coding variants
  * [dBNSFP v3.5](https://sites.google.com/site/jpopgen/dbNSFP) - database of non-synonymous functional predictions (August 2017)

###  Variant frequency databases
  * [gnomAD r2](http://exac.broadinstitute.org/) - germline variant frequencies exome-wide (October 2017)
  * [dbSNP b151/b150](http://www.ncbi.nlm.nih.gov/SNP/) - database of short genetic variants (October 2017)
  * [1000Genomes phase3](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/) - germline variant frequencies genome-wide (May 2013)
  * [Cancer Hotspots](http://cancerhotspots.org) - a resource for statistically significant mutations in cancer (2017)

### Variant databases of clinical utility
  * [ClinVar](http://www.ncbi.nlm.nih.gov/clinvar/) - database of clinically related variants (November 2018)

### Protein domains/functional features
  * [UniProt/SwissProt KnowledgeBase 2018_10](http://www.uniprot.org) - resource on protein sequence and functional information (November 2018)
  * [Pfam v32](http://pfam.xfam.org) - database of protein families and domains (September 2018)

### Cancer gene knowledge bases
  * [CancerMine v6](https://zenodo.org/record/1481254#.W-nRznpKiL4) - Literature-mined database of tumor suppressor genes/proto-oncogenes (November 2018)
     * We require a *minimum of three PubMed hits* to include candidate oncogenes and TS genes

### Phenotype ontologies
  * [MedGen](https://www.ncbi.nlm.nih.gov/medgen/)
